jni/processors/reverb.cpp

```diff
@@ -0,0 +1,233 @@
+/**
+ * Ported from mdaReverb.h
```

Comment:
Taken from MDA VST Plug-ins as it was the cleanest reverb implementation on offer.